app/javascript/views/usage/index.js

```diff
@@ -37,25 +37,40 @@ $(() => {
   // TODO: Most of these event listeners would not be necessary if JQuery and
   //       all other JS libraries were available to the js.erb files. Reevaluate
   //       this JS once we move to Rails 5 and properly configure webpacker
-  let drawnChart = null;
-  const monthlyPlanTemplatesChart = document.getElementById('monthly_template_plans');
+  let drawnChartByTemplate = null;
+  const monthlyPlanTemplatesChart = document.getElementById('monthly_plans_by_template');
   // Add event listeners that draw and destroy the chart
   monthlyPlanTemplatesChart.addEventListener('renderChart', (e) => {
-    drawnChart = drawHorizontalBar($('#monthly_template_plans'), e.detail);
+    drawnChartByTemplate = drawHorizontalBar($('#monthly_plans_by_template'), e.detail);
```

Comment:
such a weird mix of JQuery and regular JS. Will be happy to spend some time reevaluating all of our JS after we move to Rails 5!

Reply:
Yeah, I figured I'd just mimic the existing syntax for now and then we could refactor all down the line.